cs_api_cli/cs_api_cli.ml

```diff
@@ -37,9 +37,9 @@ let resolve_project_name ~client ~api ~project_id ~project_name =
 let rec analyze_trace ~client ~trace_id ~api ~count profile_id =
   let open Lwt.Infix in
   (let open Lwt_result.Infix in
-  Cs_api_core.build_analyze_request ~api ~trace_id ~profile_id
-  |> Cs_api_io.send_request ~client
-  >>= Cs_api_io.get_response_graphql)
+   Cs_api_core.build_analyze_request ~api ~trace_id ~profile_id
```

Comment:
Why the spaces? Ocamlformat new version?

Reply:
Yes, this is all ocamlformat.